walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -17,6 +17,20 @@
 import java.util.Collection;
 
 public final class FindMeetingQuery {
+  /**
+    Returns whether or not an event is attended by the required attendees from the Meeting Request
+    @param Event event The input event to compare against
+    @param Request request The subject meeting request
+    @return a boolean value that is true if the event and request have overlapping attendees.
+  */
+  private static boolean isAttended(Event event, MeetingRequest request) {
+    Collection<String> eventAttendees = event.getAttendees();
+    Collection<String> requestAttendees = request.getAttendees();
+    for(String attendee : eventAttendees) if(requestAttendees.contains(attendee)) return true;
```

Comment:
[per style guidelines](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace):
- add a space between `for` and `(`
- add a space between `if` and `(`

Reply:
Updated with commit c75f85a